### PR TITLE
R2 166 strip down useless models or processes from invoke ai docker image

### PR DESCRIPTION
**Pull request by vaidehiResleeve**
@rhuet-resleeve I have pushed the changes here. Could you check this PR by building the docker with these changes?

### CODE REVIEW

**Review by rhuet-resleeve — CHANGES_REQUESTED** [review]
Currently being stuck in a loop, please have a look at what is happening and please change accordingly

**Comment by vaidehiResleeve** [comment]
I have resolved the issue by copying few stuff in the docker file. Going forward we will not be requiring these changes as we will not be using their models for generation.

File: docker/Dockerfile [hunk 1]
```diff
@@ -66,8 +66,8 @@ COPY --link pyproject.toml .
 COPY --link invokeai/version/invokeai_version.py invokeai/version/__init__.py invokeai/version/
 ARG PIP_EXTRA_INDEX_URL
 ENV PIP_EXTRA_INDEX_URL ${PIP_EXTRA_INDEX_URL}
-RUN --mount=type=cache,target=${PIP_CACHE_DIR} \
-  "${APPNAME}"/bin/pip install .
+# RUN --mount=type=cache,target=${PIP_CACHE_DIR} \
```
Comment (rhuet-resleeve) [inline docker/Dockerfile:69]: don't comment, just remove, the original is available in the original invoke ai repo if need be

File: docker/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ ARG PYTHON_VERSION=3.9
 ##################
 FROM --platform=${TARGETPLATFORM} python:${PYTHON_VERSION}-slim AS python-base
 
-LABEL org.opencontainers.image.authors="mauwii@outlook.de"
+LABEL org.opencontainers.image.authors=
```
Comment (rhuet-resleeve) [inline docker/Dockerfile:9]: remove the line altogether?

File: invokeai/frontend/CLI/CLI.py [hunk 3]
```diff
@@ -69,30 +69,28 @@ def main():
...
-    print(f'>> InvokeAI runtime directory is "{Globals.root}"')
+
 
     # loading here to avoid long delays on startup
     # these two lines prevent a horrible warning message from appearing
     # when the frozen CLIP tokenizer is imported
     import transformers  # type: ignore
 
-    transformers.logging.set_verbosity_error()
+    # transformers.logging.set_verbosity_error()
```
Comment (rhuet-resleeve) [inline invokeai/frontend/CLI/CLI.py:86]: just remove the line

File: invokeai/frontend/CLI/CLI.py [hunk 4]
```diff
@@ -69,30 +69,28 @@ def main():
...
     import diffusers
 
-    diffusers.logging.set_verbosity_error()
+    # diffusers.logging.set_verbosity_error()
```
Comment (rhuet-resleeve) [inline invokeai/frontend/CLI/CLI.py:89]: just remove the line

File: invokeai/frontend/CLI/CLI.py [hunk 5]
```diff
@@ -109,7 +107,7 @@ def main():
         embedding_path = None
 
     # migrate legacy models
-    ModelManager.migrate_models()
+    # ModelManager.migrate_models()
```
Comment (rhuet-resleeve) [inline invokeai/frontend/CLI/CLI.py:110]: just remove the line and the comment above "migrate...."

File: invokeai/frontend/CLI/CLI.py [hunk 6]
```diff
@@ -151,7 +150,8 @@ def main():
 
     # preload the model
     try:
-        gen.load_model()
+        # gen.load_model()   # this part of the code loads the weights. Hence commented.
```
Comment (rhuet-resleeve) [inline invokeai/frontend/CLI/CLI.py:153]: remove the whole preload the model stuff, nothing is happening here anymore

**Review by rhuet-resleeve — CHANGES_REQUESTED** [review]
I let some comments, no need to build a new image, it is simply to have a cleaner dockerfile

File: docker/Dockerfile (hunk 1 above)
Comment (vaidehiResleeve) [reply docker/Dockerfile]: removed the commented lines.

File: docker/Dockerfile (hunk 2 above)
Comment (vaidehiResleeve) [reply docker/Dockerfile]: removed the commented lines.

File: invokeai/frontend/CLI/CLI.py (hunk 3 above)
Comment (vaidehiResleeve) [reply invokeai/frontend/CLI/CLI.py]: removed the commented lines.

File: invokeai/frontend/CLI/CLI.py (hunk 4 above)
Comment (vaidehiResleeve) [reply invokeai/frontend/CLI/CLI.py]: removed the commented lines.

File: invokeai/frontend/CLI/CLI.py (hunk 6 above)
Comment (vaidehiResleeve) [reply invokeai/frontend/CLI/CLI.py]: removed the commented lines.

File: invokeai/frontend/CLI/CLI.py [hunk 7]
```diff
@@ -1074,15 +1074,15 @@ def get_next_command(infile=None, model_name="no model") -> str:  # command stri
     return command
 
 
-def invoke_ai_web_server_loop(gen: Generate, gfpgan, codeformer, esrgan):
+# def invoke_ai_web_server_loop(gen: Generate, gfpgan, codeformer, esrgan):
```
[the remark this answers is not in the document]
Comment (vaidehiResleeve) [reply invokeai/frontend/CLI/CLI.py]: removed the commented lines.

**Comment by vaidehiResleeve** [comment]
@rhuet-resleeve I have made the changes as per your comments. Could you please have a look?

**Pull request merged**